pkgs/development/python-modules/async-dns/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, fetchpatch
+, poetry-core
+, python
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "async-dns";
+  version = "1.1.9";
+  disabled = pythonOlder "3.6";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "gera2ld";
+    repo = "async_dns";
+    rev = "v${version}";
+    sha256 = "0hrpjgbssvr95z9jdnmx853r6h6mih0w138hz301ll94mi8zpbsg";
```

Comment:
hash mismatch in fixed-output derivation '/nix/store/5pw32azgcikiv6gfrs4swzz5xhhcyqyd-source':
  wanted: sha256:0hrpjgbssvr95z9jdnmx853r6h6mih0w138hz301ll94mi8zpbsg
  got:    sha256:1z8j0s3dwcyavarhx41q75k1cmfzmwiqdh4svv3v15np26cywyag